hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineCreator.java

```diff
@@ -96,7 +96,7 @@ private void createPipelines() {
           if (scheduler.isClosed()) {
             break;
           }
-
+//          pipelineManager.scrubPipeline(type, factor);
```

Comment:
If this line is commented, the scrub will not be activated?

Reply:
Another thought is to run scrubber in a separate executor task so that scrubber operation won't affecting pipeline creator.

Reply:
Sorry for the typo. I meant to turn on scrubber here.
Resource wise, the background creation thread is triggered every 120s by default. I think it's a good interval for scrubber to clean up as well.